docs/Walkthrough.md

```diff
@@ -0,0 +1,76 @@
+## **EOS Plugin for Unity Walkthrough**
+ 
+This document is meant to serve as a guide to our EOS Plugin for Unity demo scenes. Each scene is meant to showcase the implementation of a specific feature such as peer 2 peer communication, lobbies, or custom invites.
+Each scene has a few things in common, the login page, side bar, scene selection, and debug log.
```

Comment:
I thin the first comma should be a colon "... few things in common: the login page ..."